sos/report/plugins/foreman.py

```diff
@@ -108,10 +108,6 @@ def setup(self):
             "/var/log/foreman-proxy-certs-generate*",
             "/usr/share/foreman/Gemfile*",
             "/var/log/{}*/foreman*".format(self.apachepkg),
-            "/var/log/{}*/katello-reverse-proxy_access_ssl.log*".format(
-                self.apachepkg),
-            "/var/log/{}*/katello-reverse-proxy_error_ssl.log*".format(
-                self.apachepkg),
```

Comment:
I would leave collecting the logs for a while due to legacy reasons. A new `sos` can run also on older foreman/Satellite/Capsule that still logs there.

E.g. checking (in downstream) current Capsule 6.10, it does log there. But it runs on RHEL7 where we wont probably backport this change. But.. we would have to be aware of "no backport" requirement, and for `sos` it is much easier to let collect potentially non-existing files (which is handled well) than risking we forget to collect something.